app/home/-components/hero-banner/template.hbs

```diff
@@ -27,9 +30,10 @@
                 </h4>
                 <SearchBar
                     local-class='searchBar'
+                    data-analytics-scope='Logged-out homepage {{this.version}}'
```

Comment:
Looks like this scope will be doubled -- it's already on the [wrapping `<main>`](https://github.com/CenterForOpenScience/ember-osf-web/pull/705/files#diff-6c8fdd895a226f961f14afd786ab490fR1)
```suggestion
```